DEMO.md

```diff
@@ -0,0 +1 @@
+Let's find out about [addPullRequestReview | GitHub Developer Guide](https://developer.github.com/v4/mutation/addpullrequestreview/).
```

Comment:
A comment created via [this GraphQL mutation](https://developer.github.com/v4/mutation/addpullrequestreviewcomment/).